src/lib/libwebgl.js

```diff
@@ -3273,9 +3273,9 @@ for (/**@suppress{duplicate}*/var i = 0; i <= {{{ GL_POOL_TEMP_BUFFERS_SIZE }}};
     // Remove all the layout(binding = x) directives so that they do not make
     // their way to the actual WebGL shader compiler. These regexes get quite
     // hairy, check against https://regex101.com/ when working on these.
-    source = source.replace(/layout\s*\(.*?binding\s*=\s*([-\d]+).*?\)/g, ''); // "layout(binding = 3)" -> ""
+    source = source.replace(/layout\s*\([^,]*?binding\s*=\s*([-\d]+)[^,]*?\)/g, ''); // "layout(binding = 3)" -> ""
```

Comment:
The actual fix here is to change the regex match to use `[^,]` (capture anything else than a `,` character), instead of a `.` (capture any character).

This way the first replace won't catch on `layout(binding = 3, ...)` or `layout(..., binding = 3)`

Reply:
It could be `layout(std140, binding=5)` for example.